templates/configuration/sites-enabled/tools-context.conf

```diff
@@ -64,11 +64,15 @@ server {
     location /gerrit {
         client_max_body_size 512m;
         proxy_pass  http://gerrit:8080;
+        proxy_set_header  X-Forwarded-For $remote_addr;
+        proxy_set_header  Host $host;
     }
 
     location /jenkins {
         proxy_pass http://jenkins:8080;
         proxy_set_header Host $host;
+        proxy_read_timeout  90;
+        proxy_redirect      http:// ###REDIRECT_PROTOCOL###://;
```

Comment:
Hi @mlushpenko, just curious is it okay that there is a space in the proxy_redirect? :) between "http://" and "###REDIRECT_..."

Reply:
@svscorp yes, it is. We redirect http:// to https:// (or to http, which doesn't do anything but provides possiblity to configure jenkins for http or https) Space is needed

Reply:
I tested both setups, both are working well, but please review when you have time as well